pyaedt/modeler/Primitives3D.py

```diff
@@ -814,6 +816,64 @@ def create_udm(self, udmfullname, udm_params_list, udm_library='syslib'):
         else:
             return False
 
+    @aedt_exception_handler
+    def create_spiral(self, internal_radius=10, spacing=1, faces=8, turns=10, width=2, thickness=1, elevation=0,
+                      material="copper", name=None):
+        """Create a spiral inductor from a polyline.
+
+        Parameters
+        ----------
+        internal_radius : float, optional
+            Internal starting point of spiral. Default is `10`.
+        spacing : float, optional
+            internal pitch between two turns. Default is `1`.
+        faces : int, optional
+            Number of faces per turn. Default is `8` as an octagon.
+        turns : int, optional
+            Number of turns. Default is `10`.
+        width : float, optional
+            Spiral width. Default is `2`.
+        thickness : float, optional
+            Spiral thickness. Default is `1`.
+        elevation : float, optional
+            Spiral elevation. Default is`0`.
+        material : str, optional
+            Spiral material. Default is `"copper"`.
+        name : str, optional
+            Spiral name. Default is `None`.
+
+        Returns
+        -------
+        :class:`pyaedt.modeler.Object3d.Polyline`
+            Polyline object.
+        """
+        assert internal_radius > 0, "Internal Radius must be greater than 0"
```

Comment:
@maxcapodi78 Just to avoid any `ZeroDivisionError` for dtheta.
```suggestion
        assert internal_radius > 0, "Internal Radius must be greater than 0."
        assert faces > 0, "Faces must be greater than 0."
```